eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -116,7 +117,7 @@ stages:
                   deploy:
                     steps:
                       - checkout: none
-                      - template: ../../tools/clone-buildtools/clone-buildtools.yml
+                      - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
```

Comment:
Should we be removing the clone-buildtools steps and switch to checkout?

Reply:
That seems neater to me but I wasn't sure if there was another reason that build tools was cloned.

Reply:
We just haven't moved everything away from the old clone-builds template yet. I will say that we have been hitting some bugs in the multi-checkout stuff so I'd be fine if you want to do that as a follow-up.